output/mux.go

```diff
@@ -0,0 +1,34 @@
+package output
+
+import (
+	"github.com/yarpc/crossdock/execute"
+	"github.com/yarpc/crossdock/plan"
+)
+
+type Mux struct {
+	Reporters []Reporter
```

Comment:
Optional: Can just do,

```go
type Mux []Reporter

func (r Mux) Start(...)  error {
  for _, reporter := range r {
    // ...
  }
}
```

This will allow usage like so,

```go
reporter := output.Mux{a, b, c}
```